EventFahrplan/EventFahrplan
# Integrate Crowdin translations

# Preliminary work
- [x] The Crowdin configuration file `crowdin.yml` has been prepared and tested with a Crowdin test instance.
  - See: 94007ca3f4d4f20cd4f5b0f5b9e8030bdd1fca7b
- [x] Upload and download with the Crowdin CLI tool have been tested.
- [x] Translations have been checked one by one and issues fixed.
- [x] Non-translatable strings have been excluded from languages other than the source language (English).

# Description
- Add new translations: Danish, Finnish.
- Update supported languages section in the `README.md`. 
- Add new language to the system language selection menu of the app.
- Update and extend existing translations: Italian, Russian, Swedish, Spanish, French, Dutch, Japanese, Polish, Portuguese.
- Document that Crowdin is used as a translation management platform.
- Exclude untranslated string when downloading them via the [Crowdin CLI tool](https://crowdin.github.io/crowdin-cli/).
-  Attribute all contributors. Thank you a lot!!!

# Test setup
1. Add all languages in the system settings
2. Stop the app, switch to a language in the system language selection menu of the app
3. Open the app
4. Long press a session and check the menu items
5. Visit the settings screen and check the texts
6. Visit the info screen and check the texts

# Successfully tested on
with `ccc36c3` flavor, `debug` build
- :heavy_check_mark: Pixel 7 emulator, Android 13 (API 33)

# Related
- [Translations - General questions and answers](https://github.com/EventFahrplan/EventFahrplan/discussions/464)
- [Crowdin translation updates (integration branch)](https://github.com/EventFahrplan/EventFahrplan/pull/466) (former integration branch, before switching to the CLI tool)

---

Closes #175 
Closes #325 
Closes #328 
Closes #427 
Closes #466
